util/update-manifest-releases/compiledreleasesops/compiled_releases_opsfile_test.go

```diff
@@ -64,15 +64,15 @@ var _ = Describe("UpdateCompiledReleases", func() {
   path: /releases/name=no-stemcell-section
   value:
     name: no-stemcell-section
-    sha1: 5ee0dfe1f1b9acd14c18863061268f4156c291a4
+    sha1: sha256:5ee0dfe1f1b9acd14c18863061268f4156c291a4
```

Comment:
This is actuall too short for a SHA-256 value. Can you adapt the tests with longer test values to avoid confusion between SHA-1 and SHA-256 values?